.github/workflows/sycl_gen_test_matrix.yml

```diff
@@ -24,6 +24,10 @@ on:
         type: string
         required: false
         default: ${{ github.run_id }}-${{ github.run_attempt }}
+      ref:
+        description: "Commit to use for checkout or github.sha by default"
+        type: string
+        required: false
```

Comment:
should we have github.sha as default here? and use just inputs.ref below?

Reply:
I'm not sure you can use variable like ${{github.sha}} as default value here. Only constants are possible I think.

Reply:
Looks like you are right and we can use variables in this place after https://github.com/Azure/actions/issues/2 fix

Reply:
P.S. Also noticed I used it before in uniq input param just fine :)